backend/api/prisonApi.ts

```diff
@@ -41,17 +43,29 @@ export default class PrisonApi {
   }
 
   public globalSearch(context: Context, params: GlobalSearchRequest, resultsLimit: number): Promise<PrisonerDetail[]> {
-    const { offenderNo, lastName, firstName, gender, location, dateOfBirth, includeAliases } = params
+    const {
+      offenderNo,
+      pncNumber,
+      lastName,
+      firstName,
+      gender,
+      location,
+      dateOfBirth,
+      includeAliases,
+      prioritisedMatch,
```

Comment:
would just hardcode this in the client and not pass it through each layer.. its a bit of weird implementation detail we don't need to expose to the rest of the code base.